Formula/x3270.rb

```diff
@@ -10,17 +10,18 @@ class X3270 < Formula
     sha256 "897d18de2b85172e751719687710f20b1717293b10dac669d6eb052e08d4b02a" => :yosemite
   end
 
+  option "without-x11", "Exclude x3270 (X11-based version)"
```

Comment:
If do that a build without arguments will fail (`brew install x3270`), an argument must be supplied. Is that okay?